### PR TITLE
Controllers do not stop interacting after a touch condition

### PR DESCRIPTION
**Pull request by Gusinuhe**
### Description

**Fixes**: If after a touch condition there is a disabled behavior that disables the same object that was previously touched, the controller that touched the object stops interacting (no more touch or grab), this can be reverted after the user uses that controller for teleporting or to show the UI ray controller.

### Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

Make a course where you have to touch something, the next step right after the touch condition, disable the very same object (like in CKD when the user has to touch the button on the screen), then try to touch or grab another object.

### Known issues

If the same procedure is also applied for a grab condition, the controller also gets disabled. That is another issue that will have its own ticket.

### CODE REVIEW

File: Editor/Properties/SnappablePropertyEditor.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ namespace Innoactive.CreatorEditor.XRInteraction
     /// Custom inspector for <see cref="SnappableProperty"/>, adding a button to create <see cref="Innoactive.Creator.XRInteraction.SnapZone"/>s automatically.
     /// </summary>
     [CustomEditor(typeof(SnappableProperty))]
-    public class SnappablePropertyEditor : Editor
```
Comment (aleksei-korolev) [inline Editor/Properties/SnappablePropertyEditor.cs:18]: API break, use [Obsolete] instead.

File: Runtime/Interaction/InteractableObject.cs [hunk 2]
```diff
@@ -194,33 +195,23 @@ protected override void OnDeactivate(XRBaseInteractor interactor)
...
         {
-            List<XRBaseInteractor> interactors = new List<XRBaseInteractor>(hoveringInteractors);
-            
-            if (interactors.Contains(selectingInteractor) == false)
-            {
-                interactors.Add(selectingInteractor);
-            }
-            
-            foreach (XRBaseInteractor interactor in interactors.Where(interactor => interactor != null))
+            bool wasTouchable = isTouchable, wasGrabbable = isGrabbable, wasUsable = isUsable;
```
Comment (aleksei-korolev) [inline Runtime/Interaction/InteractableObject.cs:201]: Please, break it into three separate declarations.

**Review by aleksei-korolev — CHANGES_REQUESTED** [review]

**Review by tomwim — APPROVED** [review]
What @aleksei-korolev said, I would also like to be changed.

Otherwise I tested it in VR and it solves the issue. What a subtle but nice fix 👍

**Review by Maxsiom — APPROVED** [review]
Tested and worked for me as well. But I would also like to see Aleksei's changes. 👍

**Comment by Gusinuhe** [comment]
The changes are not really that straightforward forward and there are still edge cases were that were not considered yet.
Due to a sensitive time for releasing, requested changes will be addressed in a future PR.

**Pull request merged**